src/main/java/duke/core/CommandManager.java

```diff
@@ -109,7 +124,13 @@ public static Command manageCommand(String userInput) throws DukeException {
                 throw new DukeException("update command fails. " + e.getMessage());
             }
         case "bye":
-            return new ExitCommand();
+            ExitCommand exitCommand = new ExitCommand();
+            for (Map.Entry<String, Integer> entry : cmdFreqTable.entrySet()) {
+                System.out.println("Key = " + entry.getKey());
+                System.out.println("Value = " + entry.getValue());
+                System.out.println("====================================================");
+            }
```

Comment:
This part is not belonged to commandManaging level. All these should be added in ExitCommand.execute.

Reply:
In Duke, command.isExit is executed after command.execute.
```
        while (!isExit) {
            try {
                String fullCommand = ui.readCommand();
                ui.showLine();
                Command c = CommandManager.manageCommand(fullCommand);
                c.execute(patientTaskList, taskManager, patientManager,
                        ui, patientTaskStorage, taskStorage, patientStorage);
                isExit = c.isExit();
```

Reply:
This code is just for me to print out for checking ... basically now exitcommand will not be keep track by the command frequency since i think it is pointless to keep track of exit command.

Reply:
Junit test is there to serve the purpose. It should not appear in CommandManager which only responsible to manage command.